src/neuroconv/utils/imports.py

```diff
@@ -0,0 +1,15 @@
+"""Helper functions for performing more complicated import operations."""
+import importlib
+from typing import Optional
+
+
+def safe_import(module_name: str, pypi_package_name: Optional[str] = None):
+    """Import a module if it is installed or raise a more informative installation error if not."""
+    try:
```

Comment:
What do you think of this pattern here vs the try-except block?
https://docs.python.org/3/library/importlib.html#checking-if-a-module-can-be-imported

Reply:
Certainly feels like a more modern approach.

Reply:
This could still be useful to include eventually though, we repeat this block so many times as is